src/functions/function-command/undeploy-function.ts

```diff
@@ -27,17 +26,17 @@ export async function undeployFunction(context: FunctionNode): Promise<string> {
     {
       cancellable: false,
       location: vscode.ProgressLocation.Notification,
-      title: `Undeploy Function: ${context.getName()}.`,
+      title: `Undeploy function ${context.getName()}.`,
     },
     async () => {
-      const result: CliExitData = await knExecutor.execute(FuncAPI.deleteFunc(context.getName()), process.cwd(), false);
+      const result: CliExitData = await execCmdCli.executeExec(FuncAPI.deleteFunc(context.getName()));
       if (result.error) {
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        vscode.window.showErrorMessage(`Fail undeploy Function: ${getStderrString(result.error)}`);
+        vscode.window.showErrorMessage(`Fail undeploy function: ${getStderrString(result.error)}`);
```

Comment:
`Failed to undeploy function {name} - error {err}`

Reply:
done